normandy/recipes/management/commands/update_addon_urls.py

```diff
@@ -1,8 +1,12 @@
-from urllib.parse import urlparse, urlunparse
 from django.core.management.base import BaseCommand
 from django.template.defaultfilters import pluralize
 
 from normandy.recipes.models import RecipeRevision
+from normandy.studies.models import Extension
+
+
+def get_filename_from_url(url):
+    url.split("/")[-1]
```

Comment:
Not only is this broken due to the lack of `return` but it also makes me nervous. How come the tests pass when `filename` always becomes `None` from the line `filename = get_filename_from_url(extension.xpi.url)`. 

Also, it makes me nervous. If the URL has `?foo=bar#anchor` on it we probably don't want that for a filename. It should probably be `urlparse(extension.xpi.url).pathname.split('/')[-1]` or something. (The `FileField.url` is unlikely to have that kind of stuff but perhaps we should mention it as a code comment then)

Reply:
This is indeed from writing too much rust lately. I'll improve the tests to catch this kind of issue. I think the better url parsing you proposed is also a good idea, I'll do that.